src/odemis/acq/fastem.py

```diff
@@ -286,28 +355,14 @@ def run(self):
         # translation pre-align are done in the correct location.
         self.move_stage_to_next_tile()
 
-        # Autofocus multiprobe ensures the beams are focused at the start of the ROA acquisition.
-        autofocus_multiprobe.run_autofocus(self._scanner,
-                                           self._multibeam,
-                                           self._descanner,
-                                           self._detector,
-                                           self._detector.data,
-                                           self._ccd,
-                                           self._stage)
+        exception = self.pre_calibrate()
 
-        # Image translation pre-alignment ensures the image of the multiprobe is roughly centered on the detector.
-        image_translation_pre_align.run_image_translation_pre_align(self._scanner,
-                                                                    self._multibeam,
-                                                                    self._descanner,
-                                                                    self._detector,
-                                                                    self._detector.data,
-                                                                    self._ccd)
+        if exception is not None:
```

Comment:
Can you add a comment here: "If calibration failed, don't acquire ROA" or similar